src/mcmc/sghmc.jl

```diff
@@ -37,22 +36,17 @@ Tianqi Chen, Emily Fox, & Carlos Guestrin (2014). Stochastic Gradient Hamiltonia
 Carlo. In: Proceedings of the 31st International Conference on Machine Learning
 (pp. 1683–1691).
 """
-function SGHMC(
-    space::Symbol...;
+function SGHMC(;
     learning_rate::Real,
     momentum_decay::Real,
     adtype::ADTypes.AbstractADType=Turing.DEFAULT_ADTYPE,
 )
     _learning_rate, _momentum_decay = promote(learning_rate, momentum_decay)
-    return SGHMC{typeof(adtype),space,typeof(_learning_rate)}(
+    return SGHMC{typeof(adtype),typeof(_learning_rate)}(
         _learning_rate, _momentum_decay, adtype
     )
```

Comment:
```suggestion
    return SGHMC(_learning_rate, _momentum_decay, adtype)
```

Are the type parameters needed here? It seems unnecessarily verbose